### DISCUSSION TITLE
没找到能单线程去挨个看视频的方法呀 老哥

### DISCUSSION THREAD

**Discussion by 2682856025** [liuyunfz/chaoxing_tool#67] [Q&A]
可以详细说下怎么改吗 因为现在学习通机制改了 只能单线程去看 这个做题就是自己搜题 有办法自动答题吗

**Comment by SongZihui-sudo** [comment]
没法自动答题，你可以提个改改，提个 pr 过来。




可以看下这个任务


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "liuyunfz/chaoxing_tool"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月14日(星期五) 晚上6:59
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[liuyunfz/chaoxing_tool] 没找到能单线程去挨个看视频的方法呀 老哥 (Discussion #67)





 
可以详细说下怎么改吗 因为现在学习通机制改了 只能单线程去看 这个做题就是自己搜题 有办法自动答题吗

**Reply by 2682856025** [reply]
什么都没有呀 老哥 单线程1一个视频一个视频看也不行吗

**Comment by SongZihui-sudo** [comment]
所以你可以拿去源码自己改改




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "liuyunfz/chaoxing_tool"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月14日(星期五) 晚上7:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [liuyunfz/chaoxing_tool] 没找到能单线程去挨个看视频的方法呀 老哥 (Discussion #67)





 
什么都没有呀 老哥 单线程1一个视频一个视频看也不行吗

**Comment by liuyunfz** [comment]
你好，可以参考下 #65 号issue，里面评论有指出修改方式。至于自动答题，程序暂不支持，也没有后期支持的打算，抱歉。
